src/mobile/button/style.ts

```diff
@@ -25,16 +25,26 @@ export const MOBILE_BUTTON_CSS = `
     display: none;
   }
   .kuc-mobile-button__button {
-    min-width: 100px;
-    height: 42px;
-    padding: 12px 12px;
+    min-width: var(--kuc-mobile-button-width, "100px");
+    width: var(--kuc-mobile-button-width, "auto");
```

Comment:
The default value can't be wrapped in quotes.
```suggestion
    min-width: var(--kuc-mobile-button-width, 100px);
    width: var(--kuc-mobile-button-width, auto);
```